recipes/configs/qwen2/7B_full.yaml

```diff
@@ -0,0 +1,78 @@
+# Config for multi-device full finetuning in full_finetune_distributed.py
+# using a Qwen2 7B model
+#
+# This config assumes that you've run the following command before launching
+# this run:
+#   tune download Qwen/Qwen2-7B-Instruct --output-dir /tmp/Qwen2-7B-Instruct
```

Comment:
I believe the download commands also need to add the `--ignore-patterns` flag to properly download safetensors files (this is slightly annoying but we need to do things this way for the time being)
```suggestion
#   tune download Qwen/Qwen2-7B-Instruct --output-dir /tmp/Qwen2-7B-Instruct --ignore-patterns ""
```